cfn-resources/ldap-verify/Makefile

```diff
@@ -1,11 +1,11 @@
-.PHONY: build test clean
+.PHONY: build debug clean
```

Comment:
is this intentional?

Reply:
yes, when you use PHONY with a list is to mimic the actions in Makefile, here I'm fixing the list to match the file